src/replica/replica.cpp

```diff
@@ -443,6 +443,33 @@ std::string replica::query_compact_state() const
     return _app->query_compact_state();
 }
 
+const char *manual_compaction_status_to_string(manual_compaction_status status)
+{
+    switch (status) {
+    case CompactionFinish:
+        return "CompactionFinish";
+    case CompactionRunning:
+        return "CompactionRunning";
+    case CompactionQueue:
+        return "CompactionQueue";
+    default:
+        dassert(false, "");
+    }
+    return "wrongCompactionStatus";
+}
```

Comment:
`enum` convert `string` seem only register like this: 
https://github.com/XiaoMi/rdsn/blob/v2.1.1/include/dsn/dist/replication/replication_enums.h

Reply:
`replication_enums` will add prefix `dsn::replication::` for status, I simply it to avoid adding this prefix.